CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ All notable, unreleased changes to this project will be documented in this file.
   - Deprecate `PUBLICATION_DATE` in `PageSortField`, the `PUBLISHED_AT` should be used instead.
   - Add a new column `pubished at` to export products. The new field should be used instead of `publication_date`.
 - Add an alternative API for fetching metadata - #9231 by @patrys
+- Fix for sending incorrect prices to Avatax - #9633 by @korycins
```

Comment:
This is part of 3.3 changelog now, but I guess we would like to port this fix to 3.3 as well?